### ISSUE TITLE
Transitive dependency issue

### ISSUE THREAD

**Issue by bmdub** [aerospike/aerospike-client-csharp#36]
It seems that the .NET Standard version of the client is unable to be referenced by a .NET Framework project.

Given the scenario:
- .NET Standard library A references AerospikeClient NuGet package from NuGet.org
- .NET Framework application B references .NET Standard library A
... results in a compiler warning:
```
There was a conflict between "AerospikeClient, Version=3.6.0.0, Culture=neutral, PublicKeyToken=26e01ad6884636d6" and "AerospikeClient, Version=3.6.3.0, Culture=neutral, PublicKeyToken=26e01ad6884636d6".
      "AerospikeClient, Version=3.6.0.0, Culture=neutral, PublicKeyToken=26e01ad6884636d6" was chosen because it was primary and "AerospikeClient, Version=3.6.3.0, Culture=neutral, PublicKeyToken=26e01ad6884636d6" was not.
```
... and a runtime error:
`System.IO.FileLoadException: 'Could not load file or assembly 'AerospikeClient, Version=3.6.3.0, Culture=neutral, PublicKeyToken=26e01ad6884636d6' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)'`

(Note that library A in our scenario must be .NET Standard so it can be consumed by both .NET Framework and .NET Core applications, so referencing the .NET Framework version of AerospikeClient is not an option.)

A couple of interesting bits:
- Note the "3.6.0" reference in the detailed output above.  Where did that come from?
- When I build the AerospikeClient package locally, then use that package instead, the problem disappears.  With no code changes at all.

This problem appears on every machine that I've seen.  We're using Visual Studio 2017 (although the problem also existed back when 2015 was used to build it).

The only conclusion I have is that maybe there's an issue with the environment on the machine in which the nuget package is built.

To reproduce the error:
- Create a .NET Standard library that references the AerospikeClient nuget package, and create a method that uses the client (constructs a client object for example).
- Create a .NET Framework console app that references the library above, and also calls the method, ensuring that the Aerospike client actually gets referenced.

Thanks,
Brian

**Comment by BrianNichols** [comment]
I have duplicated your scenario and also received an error.  I had to do these things to fix the problem.

1) In the Framework\AerospikeClient library, Change AssemblyVersion to be same as AssemblyFileVersion.  This change will occur in the next C# client release.
Old:
[assembly: AssemblyVersion("3.6")]
[assembly: AssemblyFileVersion("3.6.3")]
New:
[assembly: AssemblyVersion("3.6.3.1")]
[assembly: AssemblyFileVersion("3.6.3.1")]

2) In the test app, add this nuget package: 
        https://www.nuget.org/packages/System.ServiceModel.Primitives/4.4.1
   
    The idea to load this package came from here:
        https://github.com/dotnet/standard/issues/575


After creating a new local AerospikeClient package and loading it into the test library, the test app now worked for me.  Try your test library with this attached nupkg. 

[Aerospike.Client.3.6.3.1.nupkg.zip](https://github.com/aerospike/aerospike-client-csharp/files/1962382/Aerospike.Client.3.6.3.1.nupkg.zip)  (unzip after downloading)
